codebase/codebase.go

```diff
@@ -37,22 +40,41 @@ func (c *CodebaseContent) ToMap() map[string]interface{} {
 	res[BranchKey] = c.Branch
 	res[FileNameKey] = c.FileName
 	res[LineNumberKey] = c.LineNumber
+	res[CodebaseIDKey] = c.CodebaseID
 	return res
 }
 
+// validateURL makes sure Repo is valid GIT URL
+func validateURL(repo string) (string, bool) {
+	validPrefixes := []string{"https://github.com/", "git@github.com:"}
+	if strings.HasPrefix(repo, validPrefixes[0]) || strings.HasPrefix(repo, validPrefixes[1]) {
+		if strings.HasSuffix(repo, ".git") == false {
+			return repo + ".git", true
```

Comment:
Shouldn't we expect the user to add a valid URL with `git` ? I am not really convinced by this auto-complete `repo + .git`.